hotel-management/src/routes/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Routes, Route } from 'react-router-dom';
+
+import ROUTE_PATH from '@/constants/routePath';
+import {
+  HomePage,
+  CartPage,
+  EntryPage,
+  RoomDetailsPage,
+  RoomFormPage,
+  Layout,
+  ErrorPage,
+} from '@/pages/index';
+
+const AppRoutes = () => {
+  const {
+    HOME_PAGE,
+    CART_PAGE,
+    LOGIN_PAGE,
+    REGISTER_PAGE,
+    ROOM_DETAILS_PAGE,
+    ADD_ROOM_PAGE,
+    EDIT_ROOM_PAGE,
+    ERROR_PAGE,
+  } = ROUTE_PATH;
+
+  return (
+    <Routes>
+      <Route element={<Layout />}>
+        <Route path={HOME_PAGE} element={<HomePage />} />
+        <Route path={CART_PAGE} element={<CartPage />} />
+        <Route path={LOGIN_PAGE} element={<EntryPage />} />
+        <Route path={REGISTER_PAGE} element={<EntryPage />} />
+        <Route path={ADD_ROOM_PAGE} element={<RoomFormPage action="ADD" />} />
+        <Route path={EDIT_ROOM_PAGE} element={<RoomFormPage action="EDIT" />} />
```

Comment:
We can do this but, we should limit pass props here, instead create separate 2 pages then, we don't care these pages have same layout or anything yet